go.mod

```diff
@@ -4,12 +4,12 @@ go 1.20
 
 require (
 	github.com/hashicorp/go-memdb v1.3.4
-	github.com/hashicorp/terraform-plugin-framework v1.5.0
+	github.com/hashicorp/terraform-plugin-framework v1.5.1-0.20240222064911-3c7a391644c7
 	github.com/hashicorp/terraform-plugin-framework-timeouts v0.4.1
 	github.com/hashicorp/terraform-plugin-go v0.21.0
 	github.com/hashicorp/terraform-plugin-mux v0.14.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.32.0
-	github.com/hashicorp/terraform-plugin-testing v1.6.0
+	github.com/hashicorp/terraform-plugin-testing v1.6.1-0.20240124164911-4164c30db2a4
```

Comment:
Needs to be updated to latest release once available.

Includes addition of:

- hashicorp/terraform-plugin-testing#275